lib/apiservers/portlayer/restapi/handlers/task_handlers.go

```diff
@@ -74,6 +76,7 @@ func (handler *TaskHandlersImpl) JoinHandler(params tasks.JoinParams) middleware
 		Tty:       params.Config.Tty,
 		Attach:    params.Config.Attach,
 		OpenStdin: params.Config.OpenStdin,
+		RunBlock:  params.Config.Attach,
```

Comment:
Good point. This is lacking a FIXME comment and we need to fix Bind to handle Exec'ed sessions